recipes/anise_basil/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "anise_basil" %}
+{% set version="1.2.0" %}
+
+about:
+  home: https://github.com/seqan/{{ name }}
+  license_file: LICENSE
+  license: BSD3
+  summary: BASIL is a method to detect breakpoints for structural variants
+    (including insertion breakpoints) from aligned paired HTS reads in BAM
+    format. ANISE is a method for the assembly of large insertions from paired
+    reads in BAM format and a list candidate insert breakpoints as generated by
+    BASIL.
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  - url: https://github.com/holtgrewe/lemon-mirror/archive/8785a8ccaca992fa9b3baf21e837e888d696a219.tar.gz
```

Comment:
Yeah, generally I agree.  I had gone with the current develop version because it compiles for me (outside of conda), whereas the latest release does not.  The install notes in the readme have you do:

```
git checkout master
git submodule init
git submodule update --recursive
```

which I guessed suggested that the current master might fix some install issues.

That's also why I had created an issue to request a new release.  I'll see if I can build the release via circleci, since it appears linux doesn't seem to have the same issues as (my?) mac.

Incidentally, I installed an OS update that I hoped would fix things on my end, but it didn't.  I'm on 10.13.6.  Perhaps I'll upgrade the OS sometime this week.

Reply:
I updated seqan and anise_basil to use the available releases, though there doesn't appear to be a release for lemon, so I left that one as-is.  `conda build recipes/anise_basil` works.

I still cannot perform a local `circleci build`.  It complains that the compilers are not found.  So I tried doing the bootstrap method to build locally and I still get errors:

```
./bootstrap.py /tmp/miniconda
source ~/.config/bioconda/activate
bioconda-utils lint recipes config.yml --git-range master
bioconda-utils build recipes config.yml --docker --mulled-test --git-range master
...
17:05:13 BIOCONDA ERROR BUILD FAILED: the built package /tmp/miniconda/miniconda/conda-bld/osx-64/anise_basil-1.2.0-py27h38e7a2c_0.tar.bz2 cannot be found
17:05:13 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
17:05:13 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/anise_basil
```

So I'll be curious to see the result of the remote circleci build.  I was mentioning to Lance how cmake complains about not being able to find the compilers and he said he's had that problem before.  He recommended the bootstrap method.

Reply:
Thanks @hepcat72!!!